includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -60,6 +60,13 @@ abstract class WC_Stripe_UPE_Payment_Method {
 	 */
 	protected $supported_currencies;
 
+	/**
+	 * Can this payment method be refunded?
+	 *
+	 * @var array
+	 */
+	public $can_refund = true;
```

Comment:
@leonardola this works fine, I was just wondering about following the pattern for other properties: making it protected and having a public getter to retrieve it. What do you think?